README.MD

```diff
@@ -1,42 +1,258 @@
+</details>
+
+To run this please ensure you have docker-compose [installed](https://docs.docker.com/compose/install/).
+```shell script
+docker-compose up -d
 ```
 
-### Web build docker
-Build the web docker
+### View the Logs and Token
+To view the docker container logs and to find the server token you have to run the following command:
+```shell script
+docker logs <container-name|container-id>
+
+# to get the container name or id run the following
+docker ps 
+
+# eg. if the server gets started as displayed in the above section
+docker logs teaspeak-server
+
+# and if you want to follow the logs (ctrl+c to stop following)
+docker logs -f teaspeak-server
 ```
-docker build --tag teaweb:build web/build/
+
+
+## Web Image
+
+Basic instructions to build and host the TeaWeb client images yourself.
+
+### Alpine (~41.3MB, 21 Layers)
+This image is based on the official Nginx Docker image (Alpine Linux flavour)
+```shell script
+# Build the image with the latest web release
+docker build -f web/Dockerfile -t teaspeak/web:latest ./web
+
+# alternatively with a specific web version <gitHash>
+# docker build -f web/Dockerfile --build-arg WEB_VERSION=<gitHash> -t teaspeak/web:<gitHash> ./web
+
+# run the Ubuntu based server image
+docker run -d \
+  -p 80:80 -p 443:443 \
+  -v $(pwd)/certs:/etc/ssl/certs \
+  --restart=on-failure:3 --name teaspeak-web teaspeak/web:latest
 ```
 
-Execute the build script and build the newest version
+### Files and Volumes
+
+This image will auto-generate a self-signed certificate at initial boot-up, 
+unless the user provides one during from the get-go.
+This happens because an https connection is ***required*** by the web client to function.
+Should the user decide to provide their own certificates at initial boot or later, 
+then the following aspects have to be kept in mind:
+
+* the certificate and key have to have a name in these formats:
+  * the crt file: `tea_bundle.crt`
+  * the key file: `tea.key`
+* optionally the user can also provide a Diffie–Hellman parameter, 
+otherwise the file will also be generated at initial boot
+  * the Diffie–Hellman param: `dhparam.pem`
+ 
+All auto-generated files will be persisted in the `/etc/ssl/certs`  folder within the container.
+This folder, if not mounted by the user, will be force mounted into an anonym volume on the host machine.
+By using the above `docker run` command, or the provided `docker-compose` script for the web client a bind-mounted
+folder shall be used to store the `.crt`, `.key` and `.pem` files.
+
+### EnvVars and Build-args (web)
+
+| Variable | Default | Description |
+|:--------:|:-------:|:-----------:|
+|  **TZ**  | `Europe/Berlin` | Sets the timezone within the container. |
+
+| Build-Arg | Default | Description |
+|:---------:|:-------:|:-----------:|
+| **TEASPEAK_VERSION** | `9a23f96` | This build argument sets the web client version to be installed in the resulting image |
```

Comment:
When the version for the server is `SERVER_VERSION` why not use `WEB[CLIENT]_VERSION` here?
Edit: Just saw you already call the variable `WEB_VERSION`. So this doc. is invalid here.

Reply:
Ups, I updated the commands but forgot the EnvVar section :man_facepalming: